ts/features/itwallet/common/utils/itwTrialUtils.ts

```diff
@@ -0,0 +1,3 @@
+import { TrialId } from "../../../../../definitions/trial_system/TrialId";
+
+export const ITW_TRIAL_ID = "01J2GN4TA8FB6DPTAX3T3YD6M1" as TrialId;
```

Comment:
Should we use our env file for this ID? Does it make sense to you?

Reply:
Using env makes sense when you have different values for different environments. In this case I don't see any reason to have different IDs. I mean, we are free to use it in our local development server. Does this makes sense to you?

Reply:
I would argue that we are using the `.env` also for constants as there are a few entries which have the same value in both local and production. Up to you anyway, it was just a general comment on the matter as I'd rather have a place where all the consts are stored rather than a specialised file which probably will only contain a single const